app/api/probe_dock/metrics_api.rb

```diff
@@ -72,6 +84,17 @@ def count_new_tests_by_interval(start_date, interval)
 
           rel.to_a.inject({}){ |memo,data| memo[data.project_tests_by_interval.strftime('%Y-%m-%d')] = data.project_tests_count; memo }
         end
+
+        def project_version
+          # Retrieve the data from the specified version otherwise take the most recent project version to retrieve the data
+          if params[:projectVersionId].present?
+            ProjectVersion.where('api_id = ?', params[:projectVersionId]).first
```

Comment:
I would use `first!` here. I think we want this call to fail if provided with an unknown ID.